docs/data-sources/connectors.md

```diff
@@ -0,0 +1,36 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_connectors Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's documentation https://docs.twingate.com/docs/understanding-access-nodes.
+---
+
+# twingate_connectors (Data Source)
+
+Connectors provide connectivity to Remote Networks. This resource type will create the Connector in the Twingate Admin Console, but in order to successfully deploy it, you must also generate Connector tokens that authenticate the Connector with Twingate. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/understanding-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_connectors" "all" {}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Read-Only
+
+- `connectors` (List of Object) List of connectors (see [below for nested schema](#nestedatt--connectors))
+- `id` (String) The ID of this resource.
+
+<a id="nestedatt--connectors"></a>
+### Nested Schema for `connectors`
+
+Read-Only:
```

Comment:
There are no descriptions here -- please use the same descriptions based on my comments on the twingate_connector (singular) data source.

Reply:
not sure, why do you expecting description here? I didn't see it on other resources at this place

Reply:
I see that we miss descriptions for all Nested Schemas in other resources as well

if that's the issue, I can try to fork docs generator and fix it

Reply:
How much work do you think it will be to fix the docs generator? If it's more than half a day of effort, I think we should skip it for now, and I can create a separate issue to fix this later on.

Reply:
yeah, I think it can take more than half a day

Reply:
Okay! Created issue #148